jdaviz/configs/mosviz/tests/test_parsers.py

```diff
@@ -142,6 +143,14 @@ def test_niriss_parser(mosviz_helper, tmp_path):
     assert COMMENTCARD_KEY not in dc_1.meta
     assert dc_1.meta['bunit_data'] == 'MJy/sr'  # ASDF metadata
 
+    # We should be centered on the coordinates of the first data point
+    imview = mosviz_helper.app.get_viewer(mosviz_helper._default_image_viewer_reference_name)
+    x_pixcenter = (imview.state.x_max + imview.state.x_min)/2.0
+    y_pixcenter = (imview.state.y_max + imview.state.y_min)/2.0
+    viewer_center_coord = imview.layers[0].layer.coords.pixel_to_world(x_pixcenter, y_pixcenter)
+    assert_quantity_allclose(viewer_center_coord.ra.deg, dc_tab["R.A."][0])
+    assert_quantity_allclose(viewer_center_coord.dec.deg, dc_tab["Dec."][0])
```

Comment:
Don't really need quantity helper function since you are not testing quantity, but the value.
```suggestion
    assert_allclose(viewer_center_coord.ra.deg, dc_tab["R.A."][0])
    assert_allclose(viewer_center_coord.dec.deg, dc_tab["Dec."][0])
```

Reply:
Ahh, I thought it returned an astropy quantity. Good point